### PR TITLE
Rename folders and files to photon-{something} 

### PR DESCRIPTION
**Pull request by aminalhazwani**
@bwinton what do you think? I also have a couple of open questions regarding the `photon-colors/package.json`:

- if we change only the name/information architecture should we bump the version?
- should the `homepage` attribute point to the readme inside `/photon-colors` instead of the global one?

On a side note we could also create labels that we bind with the the `bugs/url` attribute.

### CODE REVIEW

**Comment by bwinton** [comment]
Seems reasonable, but needs some fixing up. 😃


> if we change only the name/information architecture should we bump the version?

Yeah, probably. We need to for the sketch stuff anyways.

> should the homepage attribute point to the readme inside /photon-colors instead of the global one?

Yes. Good idea.

> On a side note we could also create labels that we bind with the the bugs/url attribute.

Please do, and update the `package.json`!

Thanks!

**Comment by aminalhazwani** [comment]
ok @bwinton, here some updates!

- pulled changes from `master`
- bumped the version to v1.4.0 `photon-colors/package.json`
- created a `tokens: colors` label
- changed the `bugs/url` attribute to `photon-colors/package.json`
- changed `homepage` attribute to `photon-colors/package.json`

File: photon-colors/photon-colors.gpl [hunk 1]
```diff
@@ -0,0 +1,67 @@
+GIMP Palette
+Name: Photon Colors
+# Photon Colors GPL Color Palette v1.4.0
+
+255 26 217 Magenta 50
```
Comment (bwinton) [inline photon-colors/photon-colors.gpl:5]: We're picking up the GIMP colour renaming here, too? (I think that's okay, but wanted to make sure it was on purpose.)

File: photon-colors/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "author": "Blake Winton <bwinton@latte.ca>",
   "license": "MPL-2.0",
   "bugs": {
-    "url": "https://github.com/FirefoxUX/design-tokens/issues"
+    "url": "https://github.com/FirefoxUX/design-tokens/issues?q=is%3Aissue+is%3Aopen+label%3A%22tokens%3A+colors%22"
   },
-  "homepage": "https://github.com/FirefoxUX/design-tokens#readme"
+  "homepage": "https://github.com/FirefoxUX/design-tokens/photon-colors#readme"
```
Comment (bwinton) [inline photon-colors/package.json]: We should probably make these changes in the `photon-components/package.json` file, too.

**Review by bwinton — APPROVED** [review]
Mostly good, a couple of nits to fix, and then we can merge it! 🙂

File: photon-colors/photon-colors.gpl (hunk 1 above)
Comment (aminalhazwani) [reply photon-colors/photon-colors.gpl:5]: Yes, this is on purpose ☺️

File: photon-colors/package.json (hunk 2 above)
Comment (aminalhazwani) [reply photon-colors/package.json]: Good one! I also found out that the link needed to have the `/tree/master` path. I added to both `package.json` files and double-checked all the links.

File: photon-components/package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/FirefoxUX/design-tokens.git"
+    "url": "git+https://github.com/firefoxux/design-tokens.git"
```
Comment (bwinton) [inline photon-components/package.json]: I think this is wrong. `FirefoxUX` should be in uppercase.

File: photon-components/package.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
   "author": "Blake Winton <bwinton@latte.ca>",
   "license": "MPL-2.0",
   "bugs": {
-    "url": "https://github.com/FirefoxUX/design-tokens/issues"
+    "url": "https://github.com/firefoxux/design-tokens/labels/tokens%3A%20components"
```
Comment (bwinton) [inline photon-components/package.json]: No `?q=is%3Aissue+is%3Aopen+` on this one?

File: photon-colors/build.js [hunk 5]
```diff
@@ -27,31 +27,30 @@ const formats = {
...
     'formatter': (color, variant, value) => `exports.${color.toUpperCase()}_${variant} = '${value}';\n`,
     'ext': 'js'
   },
   'gimp': {
     'output': [`GIMP Palette
-Name: Firefox/Photon
-# Firefox Colors GPL Color Palette v${metadata.version}
-# ${metadata.homepage}
+Name: Photon Colors
+# Photon Colors GPL Color Palette v${metadata.version}
```
Comment (bwinton) [inline photon-colors/build.js]: Could we add the homepage back?

Comment (aminalhazwani) [reply photon-colors/build.js]: Sure thing! I removed it because during my test it was never visible and it was the only file that featured it.

**Pull request merged**